dev-docs/bidders/omnidex.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: bidder
+title: Omnidex
+description: Prebid Omnidex Bidder Adaptor
+biddercode: omnidex
+filename: omnidexBidAdapter
+userIds: britepoolId, criteo, id5Id, identityLink, liveIntentId, netId, parrableId, pubCommonId, unifiedId
+tcfeu_supported: true
```

Comment:
If you claim `tcfeu_supported`, you must provide a `gvl_id` that can be checked for consent

```suggestion
tcfeu_supported: false
```

Reply:
@saar120 could fix this issue? Then the docs are ready to merge 😄